packages/aws-rfdk/lib/deadline/lib/thinkbox-docker-images.ts

```diff
@@ -0,0 +1,224 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { randomBytes } from 'crypto';
+import * as path from 'path';
+
+import {
+  Repository,
+} from '@aws-cdk/aws-ecr';
+import {
+  ContainerImage,
+} from '@aws-cdk/aws-ecs';
+import {
+  Code,
+  SingletonFunction,
+  Runtime,
+} from '@aws-cdk/aws-lambda';
+import { RetentionDays } from '@aws-cdk/aws-logs';
+import {
+  Construct,
+  CustomResource,
+  Duration,
+  Token,
+} from '@aws-cdk/core';
+
+import {
+  IVersion,
+  RenderQueueImages,
+  ThinkboxManagedDeadlineDockerRecipes,
+  UsageBasedLicensingImages,
+} from '.';
+
+/**
+ * Interface to specify the properties when instantiating a {@link ThinkboxDockerImages} instnace.
+ */
+export interface ThinkboxDockerImagesProps {
+  /**
+   * An optional region for the ECR repository.
+   * @default global
+   */
+  readonly region?: string;
+
+  /**
+   * The Deadline version to obtain images for.
+   * @default latest
+   */
+  readonly version?: IVersion
+}
+
+/**
+ * An API for interacting with staged Deadline Docker images provided by AWS Thinkbox.
+ *
+ * This provides container images as required by RFDK's Deadline constructs such as
+ *
+ * * {@link @aws-rfdk/deadline#RenderQueue}
+ * * {@link @aws-rfdk/deadline#UsageBasedLicensing}
+ *
+ * @example Construct a RenderQueue
+ *
+ * import { App, Stack, Vpc } from '@aws-rfdk/core';
+ * import { RenderQueue, Repository, ThinkboxDockerRecipes } from '@aws-rfdk/deadline';
+ * const app = new App();
+ * const stack = new Stack(app, 'Stack');
+ * const vpc = new Vpc(app, stack);
+ * const images = new ThinkboxDockerImages(stack, 'Image'});
```

Comment:
```suggestion
 * const images = new ThinkboxDockerImages(stack, 'Image');
```

Reply:
This looks outdated and the new code looks correct - resolving.